.ruby-version

```diff
@@ -1 +1 @@
-2.1.4
+2.3.0
```

Comment:
Does .rubocop.yml need to be updated further to 2.3 as a result of upgrading the ruby version here?